database/gdb/gdb_core_underlying.go

```diff
@@ -278,10 +278,10 @@ func (c *Core) DoCommit(ctx context.Context, in DoCommitInput) (out DoCommitOutp
 		c.writeSqlToLogger(ctx, sqlObj)
 	}
 	if err != nil && err != sql.ErrNoRows {
-		err = gerror.NewCodef(
+		err = gerror.WrapCodef(
 			gcode.CodeDbOperationError,
-			"%s, %s",
-			err.Error(),
+			err,
+			"%s",
 			FormatSqlWithArgs(in.Sql, in.Args),
```

Comment:
Is it better using 
`gerror.WrapCode(gcode.CodeDbOperationError, err, FormatSqlWithArgs(in.Sql, in.Args))`
rather than 
`gerror.WrapCodef(gcode.CodeDbOperationError, err, "%s", FormatSqlWithArgs(in.Sql, in.Args))`
?

Reply:
sure

Reply:
@WesleyWu Should it use `gerror.WrapCode` here?